src/js/profile/mobile/widget/Calendar.js

```diff
@@ -0,0 +1,293 @@
+					todayMonth = todayMonth - 1;
+					if (todayMonth === 0) {
+						todayMonth = 12;
+						todayYear = todayYear - 1;
+					}
+					prototype._deleteCalendar();
+					today = new Date(todayYear, todayMonth - 1);
+					prototype._buildCalendar();
+				} else {
+					event.preventDefault();
+				}
+			};
+
+			prototype._unBindEvents = function (element) {
+				var self = this;
+
+				events.off(element, "vclick", self, false);
+			}
+
+			prototype._bindEvents = function (element) {
+				var self = this;
+
+				events.on(element, "vclick", self, false);
+			};
+
+			prototype.handleEvent = function (event) {
+				var self = this;
+
+				if (event.type === "vclick") {
+					self._onClick(event);
+				} else {
+					event.preventDefault();
+				}
+			};
+
+			prototype._refresh = function () {
+				var self = this;
+
+				self._setValue(self.options.value);
+			}
+
+			prototype._setValue = function (value) {
+				var self = this,
+					options = self.options;
+
+				options.pastSelection = value;
+			}
+
+			prototype._build = function (element) {
+				var controllerElement = document.createElement("div"),
+					leftArrowElement = document.createElement("div"),
+					rightArrowElement = document.createElement("div"),
+					viewChangeElement = document.createElement("div"),
+					viewTableElement = document.createElement("table"),
+					spaceElement = document.createElement("tr"),
+					oneWeekElement = document.createElement("tr"),
+					monElement = document.createElement("td"),
+					tueElement = document.createElement("td"),
+					wedElement = document.createElement("td"),
+					thuElement = document.createElement("td"),
+					friElement = document.createElement("td"),
+					satElement = document.createElement("td"),
+					sunElement = document.createElement("td");
+
+				// Controller
+				controllerElement.classList.add("controller");
+				leftArrowElement.classList.add("calendarArrowLeft");
+				rightArrowElement.classList.add("calendarArrowRight");
+				viewChangeElement.classList.add("calendarSwitch");
+
+				controllerElement.appendChild(leftArrowElement);
+				controllerElement.appendChild(rightArrowElement);
+				controllerElement.appendChild(viewChangeElement);
+
+				// View Container
+				viewTableElement.classList.add("calendarView");
+				spaceElement.classList.add("topSpace");
+				oneWeekElement.classList.add("oneWeek");
+
+				viewTableElement.id = "calendar";
+
+				monElement.innerHTML = days[0];
+				oneWeekElement.appendChild(monElement);
+				tueElement.innerHTML = days[1];
+				oneWeekElement.appendChild(tueElement);
+				wedElement.innerHTML = days[2];
+				oneWeekElement.appendChild(wedElement);
+				thuElement.innerHTML = days[3];
+				oneWeekElement.appendChild(thuElement);
+				friElement.innerHTML = days[4];
+				oneWeekElement.appendChild(friElement);
+				satElement.innerHTML = days[5];
+				oneWeekElement.appendChild(satElement);
+				sunElement.innerHTML = days[6];
+				sunElement.id = "sunday";
```

Comment:
remove `id` from sunElement

Reply:
sorry. id is incorrect.
class is correct.
I add sunElement.classList.add("sunday");